scraper/src/libretexts2zim/generator.py

```diff
@@ -157,11 +155,17 @@ def run(self) -> Path:
             name=self.zim_config.library_name, slug=self.libretexts_client.library_slug
         )
         formatted_config = self.zim_config.format(metadata.placeholders())
-        zim_path = Path(self.output_folder, f"{formatted_config.file_name_format}.zim")
+        zim_file_name = f"{formatted_config.file_name_format}.zim"
+        zim_path = self.output_folder / zim_file_name
+
+        if zim_path.exists():
+            if self.overwrite_existing_zim:
+                zim_path.unlink()
```

Comment:
I see my comment in other PR to be answered here 😉

Reply:
Indeed ^^